src/main/java/com/intuit/graphql/orchestrator/federation/exceptions/EmptyFieldsArgumentFederationDirective.java

```diff
@@ -0,0 +1,11 @@
+package com.intuit.graphql.orchestrator.federation.exceptions;
+
+import com.intuit.graphql.orchestrator.stitching.StitchingException;
+
+public class EmptyFieldsArgumentFederationDirective extends StitchingException {
```

Comment:
I'm thinking of only one Exception class for directive arguments, i.e. DirectiveArgumentException

Reply:
then inside this class we can have different error message templates